src/fmeval/model_runners/extractors/jumpstart_extractor.py

```diff
@@ -146,10 +147,15 @@ def get_jumpstart_sdk_manifest(region: str) -> Dict:
         jumpstart_bucket_base_url = os.environ.get(
```

Comment:
Please update the return type to `List[Dict]`.

Reply:
This static method actually returns a list of dict, hence need to change that to `List[Dict]`. Later in the flow where we are using this method, we get an item as a dict from the returned list and use that dict to get `modelId` for checking valid model id.